jolly-jellyfish/src/page_maker/models.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime
+from pathlib import Path
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.validators import validate_image_file_extension
+from django.db import models
+from django.utils import timezone
+
+
+def get_theme_path():
```

Comment:
A docstring would be nice here, too. You could argue that the purpose of this function is obvious (and you'd be right), but docstrings are more than just developer docs, they can serve an important purpose as API docs due to how `__doc__` works.

As this function seems to be used only in this file, it would also be good practice to prefix it with a leading underscore.